src/test/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/BalanceSheetServiceImplTests.java

```diff
@@ -411,6 +455,29 @@ void postMultipleYearFilerBalanceSheetCurrentPeriodFailureBadRequestWithNoValida
                                                         "0064000"));
     }
 
+    @Test
+    @DisplayName("Multiple Year Filer - POST - Balance Sheet - Current Period Failure - Throws ServiceException due to ApiErrorResponseException - Non Bad Request")
+    void postMultipleYearFilerBalanceSheetCurrentPeriodFailureNonBadRequestWithNoValidationErrors() throws ApiErrorResponseException, URIValidationException, ServiceException {
+        mockApiClient();
+        createMultipleYearFilerCompanyProfile();
+        createMultipleYearFilerSmallFullAccountPost();
+
+        BalanceSheet balanceSheet = createMultipleYearFilerBalanceSheetTestObject();
+        mockPreviousPeriodPost(balanceSheet);
+        mockCurrentPeriodPost(balanceSheet);
+
+        HttpResponseException httpResponseException = new HttpResponseException.Builder(404,"Bad Request",new HttpHeaders()).build();
```

Comment:
Should this not be a 404? Same with the others, too

Reply:
Well, it can be anything, its just something other than `400` as that is what we are testing.

Reply:
Ah sorry, just being an idiot 👍